### DISCUSSION TITLE
Security Vulnerability in mono.cecil package used in xUnit

### DISCUSSION THREAD

**Discussion by neham04** [xunit/xunit#2291] [Question] [answered]
On scanning the xUnit main branch, a security vulnerability shows up -
The TLS stack in Mono before 3.12.1 allows remote attackers to have unspecified impact via vectors related to client-side SSLv2 fallback.
Seems the mono.cecil library needs to be upgraded to version 3.12.x

**Comment by neham04** [comment, marked as answer]
Raised an issue for the same.

**Reply by bradwilson** [reply]
Closing since this was moved to an issue: https://github.com/xunit/xunit/issues/2292
